scripts/create-actions.py

```diff
@@ -192,9 +230,22 @@ def main(args):
         args.interactive,
     )
 
+    errors = []
     for markdown_file in args.actions_files:
-        processor.process_actions(markdown_file)
+        try:
+            processor.process_actions(markdown_file)
+        except NoActions:
+            errors.append("WARNING: no actions were found in {!r}".format(
+                markdown_file.name,
+            ))
+
+    if errors:
+        formatter = Formatter()
+        for message in errors:
+            print(formatter.as_error(message))
+
+    return len(errors)
```

Comment:
Fun fact: If there are more than 255 errors, it'll actually return an ok error status, because of overflows in the kernel.

Django had this exact same issue way back when.

```suggestion
    return 1 if errors else 0
```

Source: Some talk I watched ages ago, and https://unix.stackexchange.com/questions/37915/why-do-i-get-error-255-when-returning-1

Reply:
`pip install colorama`

Reply:
@trickeydan how does `colorama` help with the above? All it does is style text, right?